src/CacheEvents.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gregurco\Bundle\GuzzleBundleCachePlugin;
+
+final class CacheEvents
+{
+    /**
+     * The INVALIDATE event occurs manually.
+     *
+     * This event allows you to invalidate a response for a request.
+     *
+     * @Event("Gregurco\Bundle\GuzzleBundleCachePlugin\Event\InvalidateRequestEvent")
```

Comment:
you use `Event` here but didn't import Event class. Is it mistake or not?

Reply:
no mistake

Reply:
yep, I see in symfony examples :slightly_smiling_face: interesting...